coremltools/converters/mil/mil/ops/defs/tensor_operation.py

```diff
@@ -516,19 +530,19 @@ def type_inference(self):
 class tile(Operation):
     """
     Returns a new tensor by replicating input ``x`` multiples times.
-    The ``i``th dimention of ``x`` will be replicated ``reps[i]`` times.
+    The ``i`` dimention of ``x`` will be replicated ``reps[i]`` times.
```

Comment:
Revert the change. i-th dimension is the right lingo. Alternative way to say it is Dimension `i` of `x` etc

Reply:
Sorry, not used to the lingo, and I wanted to fix the font irregularity that occurs with `i`-th in a docstring (though it works fine here). Rewrote to:

> Dimension ``i`` of ``x`` will be replicated ``reps[i]`` times.